src/managing-workflow/production-deployments.md

```diff
@@ -1,3 +1,57 @@
 # Production Deployments
 
-TODO (bacongobbler): rewrite for v2
+Many Workflow users are running Workflow quite successfully in production. When readying a Workflow
```

Comment:
Agreed. This is definitely a little forward-thinking. It was a copy-over from the old v1 documentation.